erlang-checks/src/main/java/org/sonar/erlang/checks/NoTrailingWhitespaceCheck.java

```diff
@@ -53,8 +53,7 @@ public void visitFile(AstNode astNode) {
   }
 
   private void checkFileIndention(File source) throws FileNotFoundException {
-    Scanner scanner = new Scanner(new FileInputStream(source));
-    try {
+    try (Scanner scanner = new Scanner(new FileInputStream(source))) {
```

Comment:
Does `Scanner` auto-close?

Reply:
Yes, this is a new Java feature: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html#:~:text=The%20try%20%2Dwith%2Dresources%20statement%20is%20a%20try%20statement%20that,Any%20object%20that%20implements%20java.